src/parser/jobs/brd/modules/Sidewinder.js

```diff
@@ -108,30 +207,121 @@ export default class Sidewinder extends Module {
-						<Trans id="brd.sidewinder.list.one-dot">
-							{this._createTimelineButton(cast.timestamp)} <ActionLink {...ability}/> was cast with only <StatusLink {...getDataBy(STATUSES, 'id', cast.dotsApplied[0])}/> applied.
-						</Trans>
-					</List.Content>
-				</List.Item>
+			let singleTargetItem
+			let notBothDotsItem
+
+			if (!cast.hasBothDots) {
+				if (cast.dotsApplied) {
+					notBothDotsItem =
+							<Trans id="brd.sidewinder.list.one-dot">
+								<ActionLink {...ability}/> was cast with only one DoT applied to the target {cast.targetsHit > 1 ? 'and hit ' + cast.targetsHit + ' enemies' : ''}
+							</Trans>
+
+				} else {
+					notBothDotsItem = <Trans id="brd.sidewinder.list.no-dots">
+						<ActionLink {...ability}/> was cast with no DoTs applied to the target {cast.targetsHit > 1 ? 'and hit ' + cast.targetsHit + ' enemies' : ''}
+					</Trans>
+
+				}
 			}
+			if (cast.isSingleTargetShadowbite) {
+				singleTargetItem = <Trans id="brd.sidewinder.list.single-target-shadowbite">
+					{!cast.hasBothDots ? ' and' : <ActionLink {...ability}/>}  only hit a single enemy
+				</Trans>
+			}
+
+			totalPotencyLost += cast.missedPotency
+
 			return <List.Item key={cast.timestamp}>
-				<List.Content>
-					<Trans id="brd.sidewinder.list.no-dots">
-						{this._createTimelineButton(cast.timestamp)} <ActionLink {...ability}/> was cast with no DoTs applied.
-					</Trans>
+				{this._createTimelineButton(cast.timestamp)}
+				<List.Content verticalAlign="middle">
+					{notBothDotsItem}{singleTargetItem}. <Label horizontal size="small" color="red" pointing="left"><Trans id="brd.sidewinder.list.missed-potency"><Icon name="arrow down"/>Lost {this._formatDamageNumber(cast.missedPotency)} potency</Trans></Label>
 				</List.Content>
 			</List.Item>
 		})
 		// Output is a List, where every item is an incorrect cast
-		return <List divided relaxed>
+		return <>
+		<List divided relaxed>
 			{items}
 		</List>
+		<Message warning attached="bottom"><Trans id="brd.sidewinder.total-mistakes">{this._badCasts.length} mistake{this._badCasts.length !== 1 ? 's' : ''} lost a total of {this._formatDamageNumber(totalPotencyLost)} potency over the course of the fight</Trans></Message>
+		</>
+	}
+
+	_formatDamageNumber(damage) {
+		const truncDamage = Math.trunc(damage)
+		return truncDamage.toLocaleString()
+	}
+
+	_getRawDamage(event) {
+		let fixedMultiplier = event.debugMultiplier
+
+		// AND ALSO FOR RANGED TRAIT, BECAUSE APPARENTLY IT'S PHYSICAL DAMAGE ONLY REEEEEEEEEE
+		fixedMultiplier = Math.trunc((fixedMultiplier + TRAIT_STRENGTH) * 100) / 100
+
+		// We get the unbuffed damage
+		let rawDamage = event.amount / fixedMultiplier
+
+		// And then strip off critical hit and direct hit mods
+		if (event.criticalHit) {
+			rawDamage = Math.trunc(rawDamage / this.additionalStats.critMod)
+		}
+
+		if (event.directHit) {
+			rawDamage = Math.trunc(rawDamage / DHIT_MOD)
+		}
+		return rawDamage
+	}
+
+	_checkForSingleTargetShadowbites() {
```

Comment:
Not going to block on it; but for a function prefixed with "`check`", there's a _lot_ of logic and side-effects going on here.